61 - exposing APIs for controlled vocabularies

ginettev opened the pull request:
Co-authored-by: Keerthana <keerthana.r@thoughtworks.com>

ioggstream commented:
@ash1425 WRT API parameter names there are some hints in the guidelines:

-snake_case
- query parameters such as `limit` `offset` `cursor` `sort`

In this case, having the opportunity to check values using a yaml spec would be great :)

ginettev commented:
@ioggstream We have changed the code to use snake case for the request params now.

We have added page based pagination instead of cursor based pagination as it is easily supported by Elasticsearch. Also, specifying the page number and size seemed easier for the developers using the APIs too. We can discuss on this further if needed.
